tests/FunctionAwaitAllTest.php

```diff
@@ -34,8 +34,10 @@ public function testAwaitAllRejected()
         Block\awaitAll($all, $this->loop);
     }
 
-    public function testAwaitAllRejectedWithFalseWillWrapInUnexpectedValueException()
+    public function testAwaitAllRejectedWithExceptionWillWrapInUnexpectedValueException()
```

Comment:
It looks like the name should not have been changed as the logic didn't actually change?

Reply:
Oops - accident!